libraries/AP_Proximity/AP_Proximity_Boundary_3D.h

```diff
@@ -0,0 +1,147 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#include <AP_InternalError/AP_InternalError.h>
```

Comment:
I don't think we need this AP_InternalError include.  Worst case I think it can be moved to AP_Proximity_Boundary_3D.cpp.